organization-mgt-authz-valve/src/main/java/org/wso2/carbon/identity/organization/mgt/authz/valve/OrganizationMgtAuthzValve.java

```diff
@@ -47,6 +48,7 @@
 
 import static org.wso2.carbon.identity.auth.service.util.Constants.OAUTH2_ALLOWED_SCOPES;
 import static org.wso2.carbon.identity.auth.service.util.Constants.OAUTH2_VALIDATE_SCOPE;
+import static org.wso2.carbon.identity.organization.mgt.authz.service.util.Constants.ErrorMessage.ERROR_NOT_AUTHORIZED;
```

Comment:
Let's avoid static imports.

Reply:
Didn't change the static imports because all of the constants are imported like that